src/Mhwd.cpp

```diff
@@ -614,7 +619,7 @@ bool Mhwd::runScript(std::shared_ptr<Config> config, MHWD::TRANSACTIONTYPE opera
     for (auto&& foundDevice = foundDevices.begin();
             foundDevice != foundDevices.end(); ++foundDevice)
     {
-        bool found = false;
+        static bool found = false;
```

Comment:
That change just makes the reader confused. You stumble across it and begin to wonder why did someone make that variable static. We don't save memory (found is a bool which is a primitive type - no dynamic memory allocation), we just make the code more confusing and I don't think that's the way to go.